training/amd-bootc/Containerfile

```diff
@@ -21,13 +18,15 @@ RUN dnf install -y \
   pciutils \
   rocm-smi \
   tmux \
+  rsync \
   ${EXTRA_RPM_PACKAGES} \
   && dnf clean all
 
 # Setup /usr/lib/containers/storage as an additional store for images.
 # Remove once the base images have this set by default.
 RUN sed -i -e '/additionalimage.*/a "/usr/lib/containers/storage",' \
-        /etc/containers/storage.conf
+	/etc/containers/storage.conf && \
+	cp /run/.input/ilab /usr/local/bin/ilab
```

Comment:
This is the piece that wasn't working for us yesterday. The volume mount for the build directory wasn't working which caused this `/run/.input` to be empty. If you can confirm this works im fine with using this.

Reply:
Yeah what happened here is the following commit appears to have unintentionally deleted the parts in Makefile.common that copied the scripts into that .input:

https://github.com/containers/ai-lab-recipes/commit/7f0793570e0b0500141e3b45a17db3d8e3ac253a

Those areas are restored in this PR so brings back the correct script (the ilab script in the container can’t run on its own needs the other files in its containers, the wrapper launches a container and delegates to it)

Reply:
Wonderful thank you! cc @cooktheryan regarding our builds yesterday. Just so you are aware Jason a lot of the `quay.io/ai-lab` images got purged and never repopulated such as `quay.io/ai-lab/vllm:latest`, `quay.io/ai-lab/instructlab-intel:latest` and some others. I would love to sit down and understand which of these I need to ensure are alive and in quay.io.